src/adapters/poi/idunn_poi.js

```diff
@@ -86,28 +86,30 @@ export default class IdunnPoi extends Poi {
     }
   }
 
-  static async poiApiLoad(id, options = {}) {
+  static async poiApiLoad(obj, options = {}) {
     let rawPoi = null;
-    const url = `${serviceConfig.idunn.url}/v1/places/${id}`;
+    const url = `${serviceConfig.idunn.url}/v1/places/${obj.id}`;
     let requestParams = {};
     if (options.simple) {
       requestParams = {verbosity: 'short'};
     }
     try {
-      rawPoi = await Ajax.getLang(url, requestParams);
+      const headers = {};
+      if (obj.queryContext) {
+        obj.queryContext.fillHeaders(headers);
```

Comment:
This feature should be defined behind a feature flag, and enabled by default.  
Just in case it should be disabled because of some CORS issues.

Reply:
Yes but I won't make this check here but where we create the headers instead.

Reply:
Actually, I'll make it into the `fillHeaders` method instead. Like that no need to make the check every time we create a new `QueryContext`.